src/index.js

```diff
@@ -2,3 +2,29 @@ if (process.env.NODE_ENV === 'development') {
   const { worker } = require('./mocks/browser');
   worker.start();
 }
+
+function getPokemon(name) {
```

Comment:
This looks great. Working with `async` data is one of the more difficult things to do when starting out in JavaScript and you crushed this. 

You used one of 2 ways of waiting on data from an async function. The way you did it is called `callback notation`. The other way is through the use of the `await` keyword (often referred to as `async/await` syntax). When we have our call let's go over my solution as an example of `async/await`. Just want you to start building a mental model of both.

Very nice work on this!